utils/fasm/src/fasm.cpp

```diff
@@ -658,12 +658,14 @@ void FasmWriterVisitor::output_fasm_mux(std::string fasm_mux,
       bool root_level_connection = interconnect->parent_mode->parent_pb_type ==
           mux_input_pin->parent_node->pb_type;
 
+      auto fasm_features = vtr::join(vtr::split(mux_parts[1], ","), "\n");
```

Comment:
Do you need to strip whitespace? or want to for consistent output.

Reply:
output_fasm_features ignores whitespace (besides as a separator).